pom.xml

```diff
@@ -355,6 +355,11 @@
 <!--            <version>1.7.30</version>-->
 <!--            <scope>test</scope>-->
 <!--        </dependency>-->
+        <dependency>

```

Comment:
is this necessary?

Reply:
yes, see comment on the other pr

Reply:
please add it as `test` scope and add a comment on it in order to define why we force this resolution

Reply:
this is actually a dependancy that is already there, I'm just forcing the latest version